client/src/components/NewTicket/index.jsx

```diff
@@ -1,55 +1,27 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
+import Navbar from '../navbar/Navbar';
+import MainSidebar from '../MainSidebar';
+import NewTicketForm from './New-ticket-form';
```

Comment:
Use react naming convention for files.
```suggestion
import NewTicketForm from './NewTicketForm';
```